wnprc_billing/resources/queries/ehr_billing/miscCharges/Charges Not Billed.qview.xml

```diff
@@ -4,8 +4,17 @@
         <column name="date"/>
         <column name="project"/>
         <column name="investigator"/>
-        <column name="chargeId"/>
-        <column name="chargecategory"/>
+        <column name="chargeId/name">
+            <properties>
+                <property name="columnTitle" value="Charge Item"/>
```

Comment:
I notice we're setting this in a few .qview.xml files. Could we apply it centrally in a WNPRC .query.xml file so it catches all the ways it could be displayed instead?

Reply:
Good point, done!